Source/WebCore/dom/Document.cpp

```diff
@@ -2707,6 +2707,12 @@ VisibilityState Document::visibilityState() const
     // https://w3c.github.io/page-visibility/#visibilitystate-attribute
     if (!m_frame || !m_frame->page() || m_visibilityHiddenDueToDismissal)
         return VisibilityState::Hidden;
+#if ENABLE(PICTURE_IN_PICTURE_API)
+    if (quirks().shouldReportVisibleDueToActivePictureInPictureContent()) {
+        if (pictureInPictureElement())
```

Comment:
This can be collapsed to a single `if()`:
```suggestion
    if (quirks().shouldReportVisibleDueToActivePictureInPictureContent() && pictureInPictureElement())
```

Reply:
Will do!